Lib/test/test_unparse.py

```diff
@@ -280,8 +270,9 @@ def get_names(cls):
 
         names = []
         for d in cls.test_directories:
+            basepath = (pathlib.Path(__file__).parent / ".." / "..").resolve()
             test_dir = os.path.join(basepath, d)
-            for n in os.listdir(test_dir):
+            for n in os.listdir(test_dir): 
```

Comment:
IMHO using sole pathlib would be a good addition instead of mixing `os.path` and `pathlib`

Reply:
I agree, will change this soon